wdio.conf.js

```diff
@@ -1,4 +1,5 @@
 exports.config = {
+    hostname: 'selenium',
```

Comment:
I'm not sure of wdio config best practice so you may want to move this, but this will tell wdio which host selenium is running on (by default at port 4444).  If no value is supplied it will use `localhost`.  Localhost would point to the current container, but selenium is in a separate container.  So I've changed the hostname to reference the selenium container which is matched by line 10 in the docker-compose.yaml file.

Ideally this value would be exposed in an environment variable then it could be set to wherever during creation of the container.

Reply:
There's no best practice that I've seen here, your approach makes sense and I'm happy to go with it.